src/store/appType/whatsapp_cloud/actions.js

```diff
@@ -0,0 +1,18 @@
+import WhatsAppCloud from '@/api/appType/whatsapp_cloud';
+
+export default {
+  async getWabaId({ commit }, { params }) {
+    commit('SET_LOADING_WABA_ID', true);
+    const { data } = await WhatsAppCloud.getWabaId(params);
+    commit('SET_WPP_CLOUD_WABA_ID', data);
+    commit('SET_FETCHED_WABA_ID', true);
+    commit('SET_LOADING_WABA_ID', false);
+  },
+  async getWhatsAppPhoneNumbers({ commit }, { params }) {
+    commit('SET_LOADING_PHONE_NUMBERS', true);
+    const { data } = await WhatsAppCloud.getWhatsAppPhoneNumbers(params);
```

Comment:
You can add a `try/catch` to handle with errors and create mutations to deal with it